pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java

```diff
@@ -424,7 +458,6 @@ public static CompletableFuture<Map<String, Integer>> getAntiAffinityNamespaceOw
                     brokerToNamespaceToBundleRange) {
 
         CompletableFuture<Map<String, Integer>> antiAffinityNsBrokersResult = new CompletableFuture<>();
-
         pulsar.getPulsarResources().getLocalPolicies().getLocalPoliciesAsync(NamespaceName.get(namespaceName))
```

Comment:
Can we add a method to check the anti-affinity policy and reuse it?

Reply:
Added static methods to get anti-affinity policy.